examples/express-all-interactions/README.md

```diff
@@ -1,29 +1,42 @@
-# Express All Interacttions Example
+# Express All Interactions Example
 
 This example shows a fully functioning app using the
 [Slack Interactive Messages](https://github.com/slackapi/node-slack-interactive-messages) package.
 
 ## Setup
 
-### Create a Slack app
+### Create and Install a Slack app
 
 1. Create an app at <https://api.slack.com/apps> (don't close this window, you'll need it later).
 
+2. Once you've created the app, navigate to the "Install App" page on the sidebar. You'll see a hyperlink to add a 'permission scope' -- click on this.
+
+3. Scroll down to **Scopes**, and under the dropdown labeled **Select Permission Scopes**, select the option that says "Add slash commands and add actions to messages (and view related content)." Save your changes.
+
+4. Click on "Install App" in the sidebar, and click on the green **Install App** button. Once you've confirmed that you want to install your app, a page with an OAuth Access Token should appear.
+
+
 ### Run the code
 
 1. Choose where you want to run:
   *  Either clone this repo, navigate to this directory, and run `npm install`
   *  Or, <a href="https://glitch.com/edit/#!/remix/slack-express-all-interactions-example"><img src="https://cdn.glitch.com/2bdfb3f8-05ef-4035-a06e-2043962a3a13%2Fremix%402x.png?1513093958726" alt="remix button" aria-label="remix" height="33"></a>
 
-2. Set the following environment variables
-  *  `SLACK_VERIFICATION_TOKEN`: Available on your app's _Basic Information_ page in Slack.
-  *  `SLACK_ACCESS_TOKEN`: Available on your app's _Install App_ page in Slack (after installing on your Development Workspace).
+2. Set the following environment variables (note: you can find these parameters in the `.env.sample` file. **If on Glitch**, copy into the `🗝.env` file and fill out there; **If local**, copy into a new `.env` file, fill that out, and run `source .env`):
+  *  `SLACK_VERIFICATION_TOKEN`: Available on your app's _Basic Information_ page in Slack. Scroll down to **App Credentials** and you will see it in the textbox labeled **Verification Token**.
```

Comment:
i think this is perhaps overly verbose, but i'm also not new enough to know that for sure. do you actually think the reader won't know to look for the field called verification token?